idea-plugin/build.gradle

```diff
@@ -73,8 +74,8 @@ tasks.withType(org.jetbrains.intellij.tasks.PrepareSandboxTask) {
     dependsOn configurations.runtimeClasspath
 
     // Also pack the formatter in its own directory
-    into("${pluginName}/impl") {
-        from configurations.formatter
+    into("${name}/impl") {
+        from configurations.formatter.filter { !configurations.runtimeClasspath.contains(it) }
```

Comment:
@fawind you removed this `.filter { !configurations.runtimeClasspath.contains(it) }` in #562, do you recall why? It seems beneficial to dedup jars, no?

Reply:
If I remember correctly, I ran into issues where dependencies of the formatter where missing when loading the formatter from the impl directory.

Not sure if we have integration tests for this. Can quickly try if the intellij bundled formatter still works after this. Wouldn't be surprised if we can re-add this. I think me removing this filter was mostly a blind guess in trying to fix those issues.

Reply:
I've verified that it works locally. I was worried that I was testing the wrong path, so I renamed the `impl` directory, restarted my IDE, and it failed to find the formatter impl

Reply:
I need to revalidate using jdk16+ targets though

Reply:
Arg, sorry for the merge then!